docs/pop_format_spec.md

```diff
@@ -0,0 +1,66 @@
+# Proof of Possession (PoP) Specification
+
+## Overview
+
+The Proof of Possession (PoP) mechanism verifies that the owner of an EOTS key
+pair also controls a BABY key pair. This validation requires five essential
+attributes, which are exposed by the `PoPExport` structure.
+
+## Attributes
+
+The `PoPExport` structure is defined
+[here](https://github.com/babylonlabs-io/finality-provider/blob/cc07bcd4dc434f7095668724aad6865bffe425e0/eotsmanager/cmd/eotsd/daemon/pop.go#L36).
```

Comment:
This links to a specific commit in the same repository,
so this will not transition well to future updates.
However, doing a relative reference will probably lead to
an outdated line reference in the future.
We could just copy the structure with its comments here maybe.

Your call

Reply:
pasted the struct